web3-react/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "web3-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@coinbase/wallet-sdk": "3.3.0",
+    "@types/node": "^16.7.13",
+    "@types/react": "^18.0.0",
+    "@types/react-dom": "^18.0.0",
+    "@walletconnect/ethereum-provider": "1.8",
+    "@web3-react/coinbase-wallet": "8.0.34-beta.0",
+    "@web3-react/core": "8.0.35-beta.0",
+    "@web3-react/eip1193": "8.0.26-beta.0",
+    "@web3-react/empty": "8.0.20-beta.0",
+    "@web3-react/gnosis-safe": "8.0.7-beta.0",
+    "@web3-react/metamask": "8.0.28-beta.0",
```

Comment:
Should the example mention that all these are not required and you can pick the ones that you need? Maybe "to make it more explicit", we could purposfully exclude one connector (and say that if you need it, you could integrate/install it manually)

Reply:
Also, any reason for using `beta` versions here? While I am particularly a fan of having strict versioning (without `^` for example), I think that for examples in particular, it would make them a bit more "up to date" if they allowed some transisent updates on patch releases.

Reply:
I can call that out in the written guide! Good call

Reply:
for point #1, I can call that out in the written guide - good call!

Reply:
For `beta`, I just followed suite with the interface, but could change it here for sure

Reply:
Plus one on the strict versioning